src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/Html/DefaultHtmlGenerator.cs

```diff
@@ -898,11 +908,11 @@ internal static string EvalString(ViewContext viewContext, string key, string fo
         /// <remarks>
         /// Not used directly in HtmlHelper. Exposed for use in DefaultDisplayTemplates.
         /// </remarks>
-        internal static TagBuilder GenerateOption(SelectListItem item, string encodedText, IHtmlEncoder htmlEncoder)
+        internal static TagBuilder GenerateOption(SelectListItem item, IHtmlContent encodedContent)
```

Comment:
Is the `encodedContent` always encoded? Does this method or the `TagBuilder` require the content to be encoded? If not, name parameter `content`.